_tutorials-ES/103_chords/103_chords.json

```diff
@@ -0,0 +1,203 @@
+{
+    "steps": [
+        {
+            "label":"Bienvenido a MEI",
+            "descFile": "103_chords_step-00-desc.html"
+        },
+        {
+            "label":"Preparation",
```

Comment:
All the labels in this file need translation?

Reply:
I thought that label had a special meaning and had not to be translated ;)